docs/conf/eu/2017/news/announcing-presentations-ticket-update.rst

```diff
@@ -0,0 +1,63 @@
+* Daniel D. Beck - `Deprecate and destroy: documenting your software’s last days </conf/eu/2017/speakers/#speaker-daniel-d-beck>`_
+* Daniele Procida - `The four kinds of documentation, and why you need to understand what they are </conf/eu/2017/speakers/#speaker-daniele-procida>`_
+* Garen Torikian - `Finally! Trustworthy and Sensible API Documentation with GraphQL  </conf/eu/2017/speakers/#speaker-garen-torikian>`_
+* Kate Wilcox - `Aw Snap!  The Docs, They Are A-Changin’ (with apologies to Bob Dylan) </conf/eu/2017/speakers/#speaker-kate-wilcox>`_
+* Lauri Apple - `Telling a Great Story on GitHub </conf/eu/2017/speakers/#speaker-lauri-apple>`_
+* Lesia Zasadna - `Requirements that you didn't know were there </conf/eu/2017/speakers/#speaker-lesia-zasadna>`_
+* Meike Chabowski - `An Alien Looking From the Outside In: Main Takeaways After One Year in Documentation </conf/eu/2017/speakers/#speaker-meike-chabowski>`_
+* Raphael Pierzina - `Hi, my name is README! </conf/eu/2017/speakers/#speaker-raphael-pierzina>`_
+* Ruthie BenDor - `Even Naming This Talk Is Hard </conf/eu/2017/speakers/#speaker-ruthie-bendor>`_
+* Stuart Culshaw - `Tech Writers Without Borders: Making the world a better place, one (numbered) step at a time </conf/eu/2017/speakers/#speaker-stuart-culshaw>`_
+* Thomas Parisot - `Writing a book in 2017  </conf/eu/2017/speakers/#speaker-thomas-parisot>`_
+* Tim Rogers - `"You never get a second chance to make a first impression": writing great "getting started" documentation </conf/eu/2017/speakers/#speaker-tim-rogers>`_
+
+Unconference Sessions
+---------------------
+
+* Ed Stenson - `One-on-One 101 </conf/eu/2017/speakers/#speaker-ed-stenson>`_
+* Floriana Pagano - `Managing community-driven documentation - how to herd doc-writing cats through motivation and a pinch of automation </conf/eu/2017/speakers/#speaker-floriana-pagano>`_
+* Kathleen De Roo - `Manage your docs like an archivist </conf/eu/2017/speakers/#speaker-kathleen-de-roo>`_
+
+Important Ticket Pricing Update
+-------------------------------
+
+**The short version**: Based on community feedback, we have adjusted our ticket pricing to be more accessible and inclusive to our European community, and refunded the difference to those who already purchased tickets.
+You can check out the new ticket pricing on our `Tickets <http://www.writethedocs.org/conf/eu/2017/tickets/>`_ page.
+
+**The long version**: Each year we examine our ticket pricing structure and determine the different tiers based on factors like production costs, attendee experience, and sponsorship fund offset.
+Even though we needed to make adjustments over the years, our goal has always been to maintain a lower price point and therefore a lower entry barrier to our events, so that documentarians from all walks of life can attend.
+
+This year we determined a ticket increase based on many factors, but it turns out that our calculations were simply wrong. There, we said it.
```

Comment:
How would you explain it? We basically aligned the prices to match more the NA conf without more careful consideration of regional aspects and income levels in Europe. Is that something that we want to specifically say?